### ISSUE TITLE
[Project]Looking for the Hipster-Surge

### ISSUE THREAD

**Issue by pasiegrist** [jsoma/data-studio-projects#195]
# Pitch

## Summary

Suddenly it is everywhere: Quinoa, coconut-Oil, avocado, hummus, almonds and so on. For a few years now, several food-trends popped up in Switzerland - especially young, urban people changed their diet and became vegan for example. Looking at the import-statistics of Switzerland I want to figure out, what food is trending and how much.

## Details

**Possible headline(s): How Hipsters impacted the Swiss-Important-Statistics

**Data set(s):** 
Federal Customs Administration
https://www.gate.ezv.admin.ch/swissimpex/public/bereiche/waren/query.xhtml

**Code repository:**
https://github.com/pasiegrist/Datastudio/blob/master/Code/02_hipster_import/How%20our%20diet%20changed%20the%20numbers.ipynb
	
**Possible problems/fears/questions:** 
The first steps with the data took me much longer than expected. The downloading from the website was a pain (there is no API, scraping wouldn't be much faster, as the website is so slow and selenium wouldn't have liked that). And then I stumbled in a few problems calculation the indexed change for the goods. 
Fears: What other unexpected problems will I run into?
Question or fear: I haven't decided yet how I want to explore the data. 

## Work so far

I downloaded the data, merged the dataframes and calculated the change for all the goods. 

As the first data steps took longer than expected I have no visualisations yet. But I want to end up with something like this.

![image](https://user-images.githubusercontent.com/15381149/43105062-8a862fdc-8ea1-11e8-9e0c-5ff28813c12b.png)

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far

**Comment by pasiegrist** [comment]
# Update
	
## Your project content: images/words/etc
	
Suddenly it is everywhere: Quinoa, coconut-oil, avocado, almonds and so on. For a few years now, several food-trends popped up in Switzerland - especially young, urban people changed their diet and became vegan for example. Looking at the import-statistics of Switzerland I want to figure out, what food is trending and how much.

As expected, avocado is going through the roof. Surprisingly the nut import has not changed a lot, the kale had a little rise. Before 2012 quinoa wasn't seperately in the statistics. Since 2012 it is now. And the imports have grown a lot – from around 700 tons a year to over 3000. 

I only edited one graph in illustrator.

![hipster_import](https://user-images.githubusercontent.com/15381149/43289610-80efd8b0-90f9-11e8-8465-113bca2a9648.png)

I was surprised that the import of coconut-oil hardly changed. But the use of this is probably too nichey and has not gotten mainstream enough.

Here is the change indexed as percentage change compared to 2007 and in Swiss Francs.

<img width="473" alt="screen shot 2018-07-26 at 17 32 13" src="https://user-images.githubusercontent.com/15381149/43289775-ff806ae6-90f9-11e8-91b3-699d5d0384ab.png">
<img width="493" alt="screen shot 2018-07-26 at 17 32 28" src="https://user-images.githubusercontent.com/15381149/43289776-ff93a6f6-90f9-11e8-8f53-a6ce21e9a6e4.png">


## Any changes in direction or topic?

Nope.

## Problems/Questions
  
Any inputs for what else I could look?
How could I graph it more appealing?


## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by benbitoun** [comment]
Even though you have it in the subhead I would still add a little t (tons) and Mio. CHF on one number of the y-axis. I look at the chart and see naked numbers and then I have to go read something in order to figure it out.

Because some of the lines are crossing it is sometimes hard to follow them. There not too many - I would add some color in order to make them more distinguishable.

- Avocado is an obvious case. But the others not so much. So is the rise due to a boom or is it just a rice because - you know - we just import more stuff than we used to (Trade Agreements, more people etc.)? Maybe have a look at the rice imports. A possible question would be: Is Quinoa taking over the Rice's place on Swiss platters? Quinoa kills the profit of Rice farmers? Etc.

Great topic, lots of possibilities

**Comment by cfelke** [comment]
Haha, I like it, that's a great topic!

I agree with Ben that different colours would be helpful to distinguish the different products from another and see the evolution of each one at first glance. Moreover, labels would be very helpful in your second and third graph. I don't know what each line represents. 
Regarding your first graph you used an apostrophe instead of a comma to separate the numbers (14'000) and I'd dish the absolute numbers in your third graph to Mio to make it more readable.

**Comment by castorsia** [comment]
Who knew trade data could be so interesting. Great topic!

As the people before me said, you need labels in your product graphs-I guess you are already on it, as well as on getting rid of the extra '000' in your y label. 

It would also be interesting, when there's a specific peak in imports, to do a google search for, I dunno, new hipster hangout opening, and then provide little annotations. 'In 2011 a lot of new artisanal cafes opened, so next year the imports of X skyrocketed', something along those lines.

Avidly awaiting the next instalment!

**Comment by pasiegrist** [comment]
# Update
	
## Your project content: images/words/etc

Suddenly it is everywhere: Quinoa, coconut-oil, avocado, kale and so on. For a few years now, several food-trends popped up in Switzerland – especially young, urban people changed their diet and became vegan for example. Looking at the import-statistics of Switzerland I want to figure out, what food is trending and how much.

### The Rise of Hipster-Food
_Indexed change of imports per year_
![template_index_size_b](https://user-images.githubusercontent.com/15381149/43481282-80c5a47c-94d3-11e8-9088-3ba9cab5db53.png)

As expected, avocado is going through the roof. Surprisingly the nut import has not changed a lot, the kale had a little rise. Before 2012 quinoa wasn't separately in the statistics. Since 2012 it is. And the imports have grown a lot – from around 700 tons a year to over 3000.	

### Google-Trend in Switzerland for Vegan, Avocado and Quinoa
_Indexed search interest for these topics in Switzerland_
<img width="878" alt="screen shot 2018-07-31 at 14 56 19" src="https://user-images.githubusercontent.com/15381149/43480711-072248c4-94d2-11e8-8b2a-2b9adfc43113.png">
_Let's see if Soma or some other person is yelling at me for using Google Trend *duckandcover*._

As quinoa and avocado imports have increased the most of my sample here, let's have a closer look on where they come from.

### Where the avocados in Switzerland come from
_Quantity in tons per year per country_
<img width="1079" alt="screen shot 2018-07-31 at 14 26 51" src="https://user-images.githubusercontent.com/15381149/43481094-f8d04586-94d2-11e8-9721-bef710841fdb.png">

## Any changes in direction or topic?
  
Nope.
	
## Problems/Questions

I want to do an area chart for quinoa as well. Do you like the one for the avocado? Is it worth it?

## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by pasiegrist** [comment]
# Final

## Project visuals/text

Suddenly it was everywhere: quinoa, coconut-oil, avocado, kale and so on. For a few years now, several food-trends popped up in Switzerland – especially young, urban people changed their diet, a lot became vegan, others just experimented with new fats and new vegetarian cuisine. This change one cannot just observe in the menus of restaurants but also in the import statistics of the Federal Customs Administration.

## Google Trend: Avocado, Quinoa and Vegan Become Popular
_Indexed search interest for these topics in Switzerland_
![trend_def_size_b](https://user-images.githubusercontent.com/15381149/43657596-4e736b1e-9724-11e8-9c20-a686c8949d0f.png)

Surprisingliy the import of nuts, kale and coconut-oil hasn't changed as much as excpected. Maybe these trends have not become mainstream enough, in the case of coconut-oil, or the import numbers were already on a high level, like the nuts and the kale and his friends.

##The Rise of Hipster-Food
_Indexed change of quantity imported per year_
![index_size_b](https://user-images.githubusercontent.com/15381149/43657658-79b15002-9724-11e8-8b8b-b21e4f208562.png)


Avocado and Quinoa imports are going through the roof. Eventhough it is not surprising, it is impressive how much imports have increased in the past few years: they have doubled for avocado since 2007 and quadruplet for quinoa since 2012. 

##The Increase of Avocado-Imports
_Imported qunatity per year in tons speified by country_
![avocado_def_size_c](https://user-images.githubusercontent.com/15381149/43657934-34a996da-9725-11e8-866d-1e5b41f5e878.png)


Before 2012 quinoa wasn't separately in the statistics. Since 2012 it is. And the imports have grown a lot – from around 700 tons a year to over 3000. Most of the quinoa comes from Bolivia, around three quarters. The second biggerst importer is Peru. 

##Quinoa becomes popular
_Imported qunatity per year in tons speified by country_
![quinoa_def_size_c](https://user-images.githubusercontent.com/15381149/43657739-a8fa7f64-9724-11e8-818a-87b5c14250ef.png)



## Details

**Headline:** 
How Hipsters Changed the Stats

**Published website version:**
https://pasiegrist.github.io/hipster/

**Code repository:**
https://github.com/pasiegrist/Datastudio/tree/master/Code/02_hipster_import

**Final data set(s):** 
https://github.com/pasiegrist/Datastudio/tree/master/Code/02_hipster_import

## What did you find to be the most difficult part of this project?
- Finding the right commodities
- The designing part

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
I am kind of satisfied with the graph. To be fully satisfied, I would do more reporting and more digging in the numbers. Maybe I would find other nice food-trends. And it would have been nice, if I found a food that is declining because of these trends. 

## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
